### ISSUE TITLE
MQTT Monitoring

### ISSUE THREAD

**Issue by zdzichu** [TwiN/gatus#803]
### Describe the feature request

Have you thought about adding MQTT monitoring? I'd like to be informed when specific topic stops receiving data (i.e. last publish was more than X minutes ago).

### Why do you personally want this feature to be implemented?

I have an IoT device which sometime hang and need to be manually reset.

### How long have you been using this project?

3 years

**Comment by TwiN** [comment]
This is my first time hearing about MQTT, but sure! In the best of worlds, Gatus would support all protocols.

**Comment by kadaan** [comment]
#596 seems like an idea way to add support for things like MQTT.  If the adoption of a custom check grows, then it could be an ideal candidate to fold into the core Gatus codebase.

**Comment by eerison** [comment]
hey @zdzichu 

just to know, do you want to monitor zigbee devices?

**Comment by zdzichu** [comment]
@eerison nope, I have no zigbee devices. Just couple ESPEasy controllers and scripts publishing to common MQTT broker.
I was looking for generic MQTT alerting in Gatus.
